eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/ShelleyMA/Serialisation/Golden/Encoding.hs

```diff
@@ -245,7 +245,7 @@ goldenEncodingTestsAllegra =
                 (ValidityInterval SNothing SNothing)
                 SNothing
                 SNothing
-                (Coin 0)
+                mempty
```

Comment:
Did we need to change this. In the Allegra Era the value is a Coin?

Reply:
I think `Coin` worked when mint was defined to be of type `Core.Value`, and the whole MaryOrAllegra stuff was kicking in and accepting `Coin` for Allegra and `MaryValue` for Mary.  But with mint as MultiAsset this doesn't apply anymore, so Coin can't be passed as a mint , because the `MATxBody` constructor changed to take a `MultiAsset` instead of a `Value`. 
However, per your previous comments, I can change this to (MultiAsset Map.empty) instead -  maybe it makes it clearer (and more efficient).